clinica/iotools/converters/adni_to_bids/adni_modalities/adni_t1.py

```diff
@@ -57,8 +58,6 @@ def convert_adni_t1(
     cprint(
         f"Calculating paths of T1 images. Output will be stored in {conversion_dir}."
     )
-    images = compute_t1_paths(source_dir, csv_dir, subjects, conversion_dir)
```

Comment:
I'm not sure to understand why you need to remove this. Actually, I wouldn't expect any change to this file.

Reply:
That should not have been removed, I don't know how it happened but it may have been related to debugging, sorry for that